pnr/openroad/meta.yaml

```diff
@@ -0,0 +1,59 @@
+# Use `conda-build-prepare` before building for a better version string.
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('openroad-', '')|replace('-', '') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+
+package:
+  name: openroad
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/The-OpenROAD-Project/OpenROAD.git
+    git_rev: master
+    patches:
+      - odb-drt-disable-tests.patch
+      - sta-add-install-options.patch
+      - gui-find-qt-only-if-enabled.patch
+
+build:
+  # number: 202202031935
+  number: {{ environ.get('DATE_NUM') }}
+  # string: 20220203_1935
+  string: {{ environ.get('DATE_STR') }}
+  script_env:
+    - CI
+
+requirements:
+  build:
+    - cmake
+    - swig
+    - bison
+    - doxygen
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - boost
+    - eigen
+    - lemon
+    - spdlog
+    - readline
+  run:
+    - python
```

Comment:
Can the tool be run with any Python version after it's built? If not, it will be necessary to restrict its version and also perhaps add a few jobs for each Python version which is going to be supported as it's done with the `syn/yosys` package.

Reply:
actually `python` shouldn't be required as a `run` dependency as the python interpreter is embedded.

Reply:
ah, but we probably still need it for being able to resolve libpython at runtime.

Reply:
trying to pin the version and see if it helps.